lunes_cms/static/js/toggle_plural_field.js

```diff
@@ -19,3 +21,67 @@ $(document).ready(() => {
   );
   $("#id_word_type").trigger("change");
 });
+
+
+
+function show_duplicates(data, parent) {
+  var existingMessage = document.getElementById("result");
+
+  if (existingMessage) {
+    existingMessage.remove();
+  }
+  
+
+  result = document.createElement("div");
+  result.setAttribute("id", "result");
+
+
+  if (data["word"]) {
+    // If there is a duplicated word, use orange background
+    result.style.backgroundColor = "orange";
```

Comment:
```suggestion
    result.style.backgroundColor = "ffbb4a";
```
I think we should stick to the [Lunes colors](https://nextcloud.tuerantuer.org/index.php/f/2543381) :)